### PR TITLE
ADD autolabeler.yml for https://github.com/probot/autolabeler

### PR DESCRIPTION
**Pull request by ndelangen**
This adds https://github.com/probot/autolabeler to this repo.

This bot adds labels to PRs, very useful for our changelog @shilman

### CODE REVIEW

File: .github/autolabeler.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'accessibility': []
```
Comment (Hypnosphi) [inline .github/autolabeler.yml]: If we need to list tags with empty patterns for some reason, you missed `patch`

File: .github/autolabeler.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+'addon: a11y': ["addons/a11y/**"]
+'addon: actions': ["addons/actions/**"]
+'addon: backgrounds': ["addons/backgrounds/**"]
+'addon: centered': ["addons/centered/**"]
+'addon: events ': ["addons/events/**"]
```
Comment (igor-dv) [inline .github/autolabeler.yml]: idk if it's critical, but there are few entries with the extra space at the end

File: .github/autolabeler.yml (hunk 1 above)
Comment (ndelangen) [reply .github/autolabeler.yml]: no we don't I just extracted the first page worth of tags, I'll clean up the empty ones and the spaces @igor-dv mentioned

File: .github/autolabeler.yml (hunk 2 above)
Comment (Hypnosphi) [reply .github/autolabeler.yml]: I wonder if there is some kind of yamllint =D

**Comment by ndelangen** [comment]
Should be good now right?

File: .github/autolabeler.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+'addon: viewport': ["addons/viewport/**"]
+'app: angular': ["app/angular/**"]
+'app: polymer ': ["app/polymer/**"]
+'app: react-native': ["app/react-native/**"]
+'app: react': ["app/react/**"]
+'app: vue': ["app/vue/**"]
+'babel / webpack': ["webpack", "babel"]
+'cli': ["lib/cli/**"]
+'compatibility with other tools': []
+'documentation': ["docs", "*.md"]
```
Comment (igor-dv) [inline .github/autolabeler.yml]: Would you consider adding things to any of `example/*` as a documentation change?  Or changing things in `core` / linter files / as a maintenance?

Comment (ndelangen) [reply .github/autolabeler.yml]: Shall we merge and add details later?

**Review by Hypnosphi — APPROVED** [review]

**Pull request merged**